src/main/java/duke/exception/Exceptions.java

```diff
@@ -69,5 +69,13 @@ public String toString() {
         public String toString() {
             return "    OOPS!!! Your time format seems incorrect, please follow yyyy-mm-dd";
         }
+    },
+
+    FINDDETAILMISSINGEXCEPTION {
+        @Override
+        public String toString() {
+            return "    OOPS!!! So what would you like to find?\n"
+                    + "    Try to add something behind 'find'.";
```

Comment:
You could combine lines 77 and 78. There is no need to separate them

Reply:
Thank you very much for pointing out my errors!!!
I have tried my best to make improvement and made it as my latest commit.